<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>690</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>808</width>
    <height>690</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QKD</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="InitBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Init</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RunSelfTestBut">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>RunSelfTest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetLaserStateBut">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>GetLaserState</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetLaserPowerBut">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>GetLaserPower</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetTimeoutBut">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>GetTimeout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetErrorCodeBut">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>GetErrorCode</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="CommandConsole">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>331</width>
      <height>541</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="autoFormatting">
     <set>QTextEdit::AutoAll</set>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="markdown">
     <string/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="overwriteMode">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="GetStartPlatesAnglesBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>GetStartPlatesAngles</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetCurPlatesAnglesBut">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>GetCurPlatesAngles</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetSignalLevelBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>GetSignalLevel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetMaxLaserPowerBut">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>GetMaxLaserPower</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetRotateStepBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>GetRotateStep</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetLightNoisesBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>GetLightNoises</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetStartLightNoisesBut">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>GetStartLightNoises</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetMaxSignalLevel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>GetMaxSignalLevel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetLaserStateBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>SetLaserState</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetLaserPowerBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>SetLaserPower</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetTimeoutBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>SetTimeout</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetPlateAngleBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>SetPlateAngle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendMessageBut">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>560</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>SendMessage</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="LaserState">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>92</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>On/Off</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="LaserPowerValue">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="TimeoutValue">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="PlateAngleValue">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="PlateNumber">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>41</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="InitAngles1">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>411</y>
      <width>125</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="InitAngles2">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>440</y>
      <width>125</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="InitAngles3">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>469</y>
      <width>125</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="InitAngles4">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>498</y>
      <width>125</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="InitByButtons">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>InitByButtons</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReadIni">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>ReadIni</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WriteIni">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>490</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>WriteIni</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InitByPD">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>InitByPD</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
